uint8 bin_id
#move to this location
bool move_near_human
